src/pages/Application.jsx

```diff
@@ -13,6 +13,16 @@ class _Application extends Component {
     dispatch(getApplicationsAction());
   }
 
+  applicationStarted = (id) => {
+    const applications = this.props.applications || [];
+    for(let i = 0; i < applications.length; i++) {
+      if(applications[i].application._id === id) {
+        return true;
+      }
+    }
+    return false;
+  };
+
```

Comment:
Any computation in the render method should be memoized using selectors

Reply:
sounds good, I will fix that.

Reply:
ummm, how do you do that though?
because I want to change component layout based on whether or not it is started. I don't want to simply not render it.

Reply:
I'll change it now

Reply:
Changed, see fab796b